docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst

```diff
@@ -110,7 +110,7 @@ We will add the following code to the file.
   - name: Test for new_option, create new user WITHOUT the attribute
     community.postgresql.postgresql_user:
       name: test_user
-      add_attribute: false
+      
```

Comment:
I would say do these changes in separate PR.

Reply:
Okay